merlin/models/tf/models/benchmark.py

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from typing import List, Optional, Union
+
+from merlin.schema import Schema
+
+from ..blocks.core.aggregation import ElementWiseMultiply
+from ..blocks.mlp import MLPBlock
+from ..blocks.retrieval.matrix_factorization import MatrixFactorizationBlock
+from ..core import Model, ParallelBlock, ParallelPredictionBlock, PredictionTask
+from .utils import parse_prediction_tasks
+
+
+def NCFModel(
+    schema: Schema,
+    embedding_dim: int,
+    mlp_block: MLPBlock,
+    prediction_tasks: Optional[
+        Union[PredictionTask, List[PredictionTask], ParallelPredictionBlock]
+    ] = None,
+    **kwargs
+) -> Model:
+    """NCF-model architecture.
```

Comment:
Should we maybe add a comment here where we link to the paper that showed that properly initialized MF can outperform NCF? cc @karlhigley